new-log-viewer/src/contexts/StateContextProvider.tsx

```diff
@@ -184,12 +189,39 @@ const StateContextProvider = ({children}: StateContextProviderProps) => {
                 updateLogEventNumInUrl(lastLogEventNum, logEventNumRef.current);
                 break;
             }
+            case WORKER_RESP_CODE.CHUNK_RESULT:
+                console.log(`[MainWorker -> Renderer] CHUNK_RESULT: ${JSON.stringify(args)}`);
+                for (const [pageNumStr, results] of Object.entries(args)) {
```

Comment:
(might be easier to refactor after we're done converting the `queryResults` into a state.)
(we might no longer write the code this way if we're to change the type of `ev.data.args` into a `Map` instead of an `Object`.

Shall we consider refactoring this code with `Object.entries.forEach(someCallbackFn)` ?